kolibri/plugins/learn/assets/src/views/ExploreLibrariesPage/index.vue

```diff
@@ -25,9 +25,9 @@
       :deviceName="device['device_name']"
       :deviceIcon="getDeviceIcon(device)"
       :channels="device.channels"
-      :totalChannels="device['total_channels']"
+      :totalChannels="device['total_count']"
```

Comment:
Good catch. Could you also please update line 54 in this way? It still uses non-existing `total_channels` property too,